### ISSUE TITLE
False positive on UnusedUsesSniff

### ISSUE THREAD

**Issue by tremby** [slevomat/coding-standard#138]
I have a `use` statement for `GuzzleHttp\Promise`. In my code I'm using a bare function from that namespace, as `Promise\settle()`. The `UnusedUsesSniff` is complaining that I'm not using `GuzzleHttp\Promise`, which I believe to be false.

I also tried `use GuzzleHttp\Promise\settle` and then using `settle()`, and I get an error from the sniff here too.

**Comment by kukulich** [comment]
@tremby Could you please post here your ruleset.xml?

**Comment by tremby** [comment]
I don't understand why, but OK.

```xml
<?xml version="1.0"?>
<ruleset name="Tribal.Laravel5">
  <description>
    Ruleset for PHP projects using Laravel 5
  </description>

  <!-- Referencing PSR2 includes the whole PSR2 standard -->
  <rule ref="PSR2"/>

  <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Arrays/TrailingArrayCommaSniff.php"/>
  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/AlphabeticallySortedUsesSniff.php"/>
  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/UnusedUsesSniff.php"/>
</ruleset>
```

**Comment by Majkl578** [comment]
True, looks like it doesn't work with functions in combination with partial uses.
Simple repro code:
```php
<?php

use GuzzleHttp\Promise;

Promise\settle($promise);
```
produces:
```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 3 | ERROR | [x] Type GuzzleHttp\Promise is not used in this file.
   |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by Majkl578** [comment]
@tremby: Could you test with  #139?

**Comment by tremby** [comment]
@Majkl578: works for me, and verified that the sniff still complains when it should (at least in trivial cases). Looking forward to a merge and tag.
